beginner/Exercise_2/Exercise_2.ipynb

```diff
@@ -151,9 +151,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# create a new branch for planning day 1\n",
-    "# use \"git switch -c <meaningful_branch_name A>\" to create a new branch\n",
-    "# (Alternative: \"git checkout -b <meaningful_branch_name A>\")\n"
+    "# create a new branch named \"planning_day1\"\n",
+    "# use \"git switch -c planning_day1\" to create a new branch\n",
+    "# (Alternative: \"git checkout -b planning_day1\")\n"
```

Comment:
Shouldn't the participants be free to use their own meaningful name for their branches?

Reply:
I think it's better if they have to do this by themselves and we don't tell them everything.